boa_engine/src/vm/opcode/mod.rs

```diff
@@ -148,6 +148,16 @@ macro_rules! generate_impl {
                 Self::INSTRUCTIONS[self as usize]
             }
 
+            const COSTS: [usize; Self::MAX] = [
+                $(<generate_impl!(name $Variant $(=> $mapping)?)>::COST),*
+            ];
+
+            /// Cycle cost of this opcode.
+            #[must_use]
+            pub const fn cost(self) -> usize {
+                Self::COSTS[self as usize]
+            }
```

Comment:
I think we should change `cost` type to something smaller like `u8`, since I don't think an Opcode should be cost more than `256`.

It would cut the cost's table size by `8` (2048 bytes to 256 bytes)